### PR TITLE
Making misscat function and test

### PR DESCRIPTION
**Pull request by EitharAlfatih**

### CODE REVIEW

File: pythcat/Untitled1.ipynb [hunk 1]
```diff
@@ -0,0 +1,175 @@
+{
```
Comment (doraqmon) [inline pythcat/Untitled1.ipynb:1]: This file is not needed. Could you please remove it from your branch?

File: pythcat/misscat.py [hunk 2]
```diff
@@ -1,16 +1,35 @@
...
-        dataframe of after dropping missing values.
-    """
\ No newline at end of file
+    dataframe of after dropping missing values.
+
+    Examples 
+    ---------
+    >>> from pythcat import misscat
+    >>> data = pd.DataFrame(data = {"X": [1, None, 2], "Y": [2, None, None], "Z": [1, 2, None]})
+    >>> misscat.misscat(data, threshold = 0.3)
```
Comment (doraqmon) [inline pythcat/misscat.py:23]: You can add the output of the example

File: pythcat/misscat.py [hunk 3]
```diff
@@ -1,16 +1,35 @@
...
+    """
+    if type(df) is not type(pd.DataFrame()):
+        raise AssertionError("Data Must be a pandas dataframe")
+    if type(threshold) is not int and type(threshold) is not float:
+        raise AssertionError("threshold must a number")
```
Comment (doraqmon) [inline pythcat/misscat.py]:
```suggestion
        raise AssertionError("threshold must be a number")
```

File: pythcat/misscat.py [hunk 4]
```diff
@@ -1,16 +1,35 @@
...
+    dataframe of after dropping missing values.
+
+    Examples 
+    ---------
+    >>> from pythcat import misscat
+    >>> data = pd.DataFrame(data = {"X": [1, None, 2], "Y": [2, None, None], "Z": [1, 2, None]})
+    >>> misscat.misscat(data, threshold = 0.3)
+    """
+    if type(df) is not type(pd.DataFrame()):
+        raise AssertionError("Data Must be a pandas dataframe")
```
Comment (doraqmon) [inline pythcat/misscat.py]:
```suggestion
        raise AssertionError("Data must be a pandas dataframe")
```

File: tests/test_misscat.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+        misscat.misscat(df = data, threshold = 2)
+
+    # Tests if an exception has been raised when a threshold less than 0 is passed
+    with pytest.raises(Exception):
+        misscat.misscat(df = data, threshold = -0.1)
+    
+    # Tests if an exception has been raised when a non numeric datatype is passed for the threshold
+    with pytest.raises(Exception):
+        misscat.misscat(df = data, threshold = "0.3")
+    data = pd.DataFrame(data = {"X": [1, None, 2], "Y": [2, None, None], "Z": [1, 2, None]})
```
Comment (doraqmon) [inline tests/test_misscat.py]: This line is not needed. We can remove it.
```suggestion

```

**Review by doraqmon — CHANGES_REQUESTED** [review]

File: pythcat/Untitled1.ipynb (hunk 1 above)
Comment (EitharAlfatih) [reply pythcat/Untitled1.ipynb:1]: I thought I did, will be deleted

**Review by doraqmon — APPROVED** [review]

**Pull request merged**